source/configuration_manual/authentication/ldap_authentication.rst

```diff
@@ -20,8 +20,13 @@ See http://wiki.dovecot.org/AuthDatabase/LDAP for more details.
     driver = ldap
   }
 
-These enable ``LDAP``to be used as ``passdb`` and ``userdb``. The userdb
+These enable ``LDAP`` to be used as ``passdb`` and ``userdb``. The userdb
```

Comment:
change passdb into 
```
:ref:`passdb <authentication-password_databases>`
```
and userdb into

```
:ref:`passdb <authentication-user-database>`
```

Reply:
userdb into  :ref:`userdb <authentication-user_database>` ?

Reply:
yes, sorry.

Reply:
will squash...